TorchSharp/TypeGeneration.tt

```diff
@@ -447,9 +448,19 @@ foreach (var type in new [] {
 
         /// <summary>
         ///  Returns the associated storage for this tensor
-        /// </summary>
-        public <#=tname#>Storage Storage => new <#=tname#>Storage (TH<#=tname#>Tensor_storage (handle));
-
+        /// </summary>       
```

Comment:
I believe [we wanted](https://github.com/xamarin/TorchSharp/pull/63#discussion_r233228263) to remove `internal storage`. Please delete the modifications to storage. Probably they will be fixed if you fetch and rebase with master.